src/tools/botPermissions.js

```diff
@@ -45,7 +45,7 @@ export class botPermissions {
   }
 
   checkBotPerms(interaction) {
-    if (this.botPerms.length && !interaction.guild.me.permissions.has(this.botPerms)) {
+    if (this.botPerms.length && !interaction.guild.members.me.permissions.has(this.botPerms)) {
```

Comment:
I get a bit of shivers when seeing the depth of `interaction.guild.members.me.permissions`. But I guess the risk of it breaking on `undefined` errors is very low right?

Reply:
Really low, there would be something really wrong with the Discord API or Discord.JS